packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -982,10 +986,35 @@ class _PopupMenuRoute<T> extends PopupRoute<T> {
   }
 }
 
-/// Show a popup menu that contains the `items` at `position`.
+/// A builder that creates a [RelativeRect] to position a popup menu.
+///
+/// The returned [RelativeRect] determines the position of the popup menu relative
+/// to the bounds of the [Navigator]'s overlay. The menu dimensions are not yet
+/// known when this callback is invoked, as they depend on the items and other
+/// properties of the menu.
+///
+/// The coordinate system used by the [RelativeRect] has its origin at the top
+/// left of the [Navigator]'s overlay. Positive y coordinates are down (below the
+/// origin), and positive x coordinates are to the right of the origin.
+///
+/// See also:
+///
+///  * [RelativeRect.fromLTRB], which creates a [RelativeRect] from left, top,
+///    right, and bottom coordinates.
+///  * [RelativeRect.fromRect], which creates a [RelativeRect] from two [Rect]s,
+///    one representing the size of the popup menu and one representing the size
+///    of the overlay.
+typedef PopupMenuPositionBuilder = RelativeRect Function();
+
+/// Shows a popup menu that contains the `items` at `position`.
 ///
 /// The `items` parameter must not be empty.
 ///
+/// Either [position] or [positionBuilder] must be provided. If both are specified,
+/// [position] takes precedence. The [positionBuilder] is called at the time the
+/// menu is shown to compute its position, which is useful when the position needs
+/// to be determined at runtime based on the current layout.
```

Comment:
Is this more correct?
```suggestion
/// [position] takes precedence. The [positionBuilder] is called at the time the
/// menu is shown to compute its position and every time the layout is updated,
/// which is useful when the position needs
/// to be determined at runtime based on the current layout.
```

Reply:
done

Reply:
Is this applied at all? I mean you can change based on my suggestion, but it seems to me that this isn't changed at all.